bench.php

```diff
@@ -1,10 +1,78 @@
 <?php
+error_reporting(E_ALL);
+ini_set ('display_startup_errors', 1);
+ini_set ('display_errors', 1);
+set_time_limit(180);
```

Comment:
I agree, but it think its vital to actually see errors (for example php-zip mod not installed/activated). Maybe you can implement @init_set().

The time limit should be increased tho, because with the default limit of 30sec. some systems will not complete all the benchmarks. Here are some numbers as an example:
my raspberry pi 2 can only do 5 benchmarks in 30sec.
my rapsberry pi 3 can only do 10 benchmarks in 30sec.
total duration on my amd 5700G/win10/xampp/php8.2: 16sec.
total duration on my all-inkl.com server: 5sec.